static/js/models/cohortdata.model.js

```diff
@@ -144,28 +153,53 @@ Cohortdata.getSummaryResults = (studyID, trait, cohort, result) => {
     })
 }
 
-Cohortdata.getFullResults = (studyID, trait, cohort, result) => {
+Cohortdata.getFullResults = (studyID, trait, cohorts, result) => {
+    sqlStatement = ""
+    snpsSqlStatement = ""
+    queryParams = []
+    snpsQueryParams = []
+    if (!Array.isArray(cohorts)) {
```

Comment:
Also consider doing null check. If cohorts is null but is put into an array, this could create problems.